serverless/aws/sam/src/main/scala/sttp/tapir/serverless/aws/sam/AwsSamOptions.scala

```diff
@@ -1,9 +1,11 @@
 package sttp.tapir.serverless.aws.sam
 
+import sttp.tapir.serverless.aws.sam.HttpApiProperties._
 import scala.concurrent.duration.{DurationInt, FiniteDuration}
 
 case class AwsSamOptions(
     namePrefix: String,
+    httpApi: Option[HttpApiProperties] = None,
```

Comment:
maybe let's add new parameters to the end of the list - otherwise code can break in a weird way (when `AwsSamOptions` was created without named parameters)